### PR TITLE
[4720] [studio] Create Workflow Management API

### PR DESCRIPTION
**Pull request by dejan-brkic**
API docs updated

### Ticket reference or full description of what's in the PR
https://github.com/craftercms/craftercms/issues/4720

### CODE REVIEW

File: src/main/api/2/studio-api-2.yaml [hunk 1]
```diff
@@ -5522,6 +5522,167 @@ paths:
...
+                  description: true if system processing is to be cleared
+                clearUserLocked:
+                  type: boolean
+                  description: true if user lock is to be cleared
+                live:
+                  type: boolean
+                  description: true if item is to be set as live, otherwise false
+                staged:
+                  type: boolean
+                  description: true if item is to be set as staged, otherwise false
```
Comment (rart) [inline src/main/api/2/studio-api-2.yaml:5608]: - Aren't we missing state here too? (i.e. to set the item(s) state)?
- Are all these JSON attributes optional? For example, we may want to just set one of them with out altering the rest.

File: src/main/api/2/studio-api-2.yaml [hunk 2]
```diff
@@ -5522,6 +5522,167 @@ paths:
...
```
Comment (rart) [inline src/main/api/2/studio-api-2.yaml]: - Where/what is the query?
- Weren't we going to have some sort of flag to apply to the entire query vs a given set of items?
- I'm wondering if these two mutating apis could be just one. They share much; we could send either query or item paths, and the backend would apply to either the entire query or the set of paths sent. All json properties - except for siteId - would be optional as they can be conjugated.

File: src/main/api/2/studio-api-2.yaml (hunk 1 above)
Comment (sumerjabri) [reply src/main/api/2/studio-api-2.yaml:5608]: @dejan-brkic please make the bits optional.

File: src/main/api/2/studio-api-2.yaml (hunk 2 above)
Comment (sumerjabri) [reply src/main/api/2/studio-api-2.yaml]: @rart the query is the set of paths and bitmap at the beginning.

File: src/main/api/2/studio-api-2.yaml [hunk 3]
```diff
@@ -5522,6 +5522,167 @@ paths:
...
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                siteId: 
+                  type: string
+                  description: site ID
+                items:
+                  type: array
```
Comment (sumerjabri) [inline src/main/api/2/studio-api-2.yaml:5593]: Why an Array here, and not at the initial query method?

**Review by sumerjabri — CHANGES_REQUESTED** [review]

File: src/main/api/2/studio-api-2.yaml (hunk 2 above)
Comment (sumerjabri) [reply src/main/api/2/studio-api-2.yaml]: @rart I don't think merging them is useful. They're quite different.

File: src/main/api/2/studio-api-2.yaml (hunk 1 above)
Comment (sumerjabri) [reply src/main/api/2/studio-api-2.yaml:5608]: @rart I think we got it all, ping me and I can explain better.

**Review by sumerjabri — APPROVED** [review]

**Pull request merged**